src/commands.rs

```diff
@@ -142,6 +146,33 @@ struct InitArgs {
     git_repo: Option<String>,
 }
 
+/// Get config options
+///
+/// Operates on jj configuration, which comes from the config file and
+/// environment variables. Uses the config file at ~/.jjconfig.toml or
+/// $XDG_CONFIG_HOME/jj/config.toml, unless overridden with the JJ_CONFIG
+/// environment variable.
+///
+/// For supported config options and more details about jj config, see
+/// https://github.com/martinvonz/jj/blob/main/docs/config.md.
+///
+/// Note: Currently only supports getting config options, but support for
+/// setting options is also planned (see
+/// https://github.com/martinvonz/jj/issues/531).
+#[derive(clap::Subcommand, Clone, Debug)]
+enum ConfigSubcommand {
+    /// Get the value for a given key.
+    Get {
+        /// The config option name
+        #[arg(required = true, value_parser=NonEmptyStringValueParser::new())]
+        name: String,
+    },
+
+    /// List all variables set in config file, along with their values.
+    #[command(visible_alias("l"))]
+    List {},
```

Comment:
Just FYI (no change required), I hope we'll be able to also say where a config value comes from once https://github.com/mehcode/config-rs/issues/319 is done. That's the same feature as `git config -l --show-origin` and `hg config --source` do.

Reply:
Oh cool, TIL about `--show-origin`.

Reply:
Went ahead and added a TODO for this to keep the link to the config-rs bug handy.